setup.py

```diff
@@ -22,7 +22,7 @@
 
 setuptools.setup(
     name="augly",
-    version="0.1.10",
+    version=open("version.txt", "r").read(),
```

Comment:
what's the reason for doing this as opposed to having the version in this file?

Reply:
The reason was since we now have the version as a str in 2 different files — it’s easier to update if it’s in one text file source of truth. I’ve seen other repos that do this.

Reply:
But @iAdityaEmpire did you check that opening the text file like this works perfectly? Just to check, since e.g. the setup.py file isn’t run in the tests.

Reply:
@zpapakipos I ran `make html` and it worked and the documentation built as well with no issues.